src/clu_main.c

```diff
@@ -33,7 +33,7 @@
  * do not want "-e" to work for encrypt, user must use "encrypt"
  */
 
-static struct option mode_options[] = {
+static const struct option mode_options[] = {
```

Comment:
Consider updating below at line 73 needs changed to const. `const struct option* current = &mode_options[I];`. And references of `current->`.